packages/eslint-plugin/src/rules/css-lint/.stylelintrc.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Let's name this file directly to include `-test`. Also please check that it does not leak to `dist` folder.